code/modules/mining/ores_coins.dm

```diff
@@ -382,37 +382,37 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 	name = "plasma coin"
 	cmineral = "plasma"
 	icon_state = "coin_plasma_heads"
-	value = 100
+	value = 40
 	materials = list(MAT_PLASMA = MINERAL_MATERIAL_AMOUNT*0.2)
 	grind_results = list("plasma" = 4)
 
 /obj/item/coin/uranium
 	name = "uranium coin"
 	cmineral = "uranium"
 	icon_state = "coin_uranium_heads"
-	value = 80
+	value = 25
 	materials = list(MAT_URANIUM = MINERAL_MATERIAL_AMOUNT*0.2)
 	grind_results = list("uranium" = 4)
 
 /obj/item/coin/bananium
 	name = "bananium coin"
 	cmineral = "bananium"
 	icon_state = "coin_bananium_heads"
-	value = 1000 //makes the clown cry
+	value = 200 //makes the clown cry
 	materials = list(MAT_BANANIUM = MINERAL_MATERIAL_AMOUNT*0.2)
 	grind_results = list("banana" = 4)
 
 /obj/item/coin/adamantine
 	name = "adamantine coin"
 	cmineral = "adamantine"
 	icon_state = "coin_adamantine_heads"
-	value = 1500
+	value = 100

```

Comment:
shouldn't this be more? the original price is 500 more than diamond

Reply:
1/5th of an individual sheet's value

Reply:
adamantine was supposed to be incredibly valuable, it was nerfed when bounties were added